resources/core/templates/tests/test-core-acceptance.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       image: {{ .Values.global.containerRegistry.path }}/{{ .Values.global.acceptance_tests.dir }}acceptance-tests:{{ .Values.global.acceptance_tests.version }}
       env:
       - name: HELM_BROKER_URL
-        value: http://{{.Release.Name }}-helm-broker.{{ .Release.Namespace }}.svc.cluster.local
+        value: http://helm-broker.kyma-system.svc.cluster.local
```

Comment:
If this is not equal to the `.Release.Namespace` anymore, maybe we could put it to global values

Reply:
According to my knowledge there is no need to keep this value as a variable. We have no scenario where we would need to override it. The global variable would have to be broker specific, something like `helm-broker-release-namespace`.